controllers/comment-controller.js

```diff
@@ -0,0 +1,14 @@
+const Comment = require('../models/Comment')
+
+exports.deleteComment = async (req, res, next) => {
+  const { id } = req.params
+
+  try {
+    const hasDeleted = await Comment.deleteById(id)
+    if (!hasDeleted) return res.sendStatus(404)
+
+    return res.sendStatus(204)
+  } catch (err) {
+    return res.sendStatus(400)
```

Comment:
This will happen for every single error. You should identify specific cases in which this is a 400 and default back to a 500 instead. 

That was if something unexpected happens, say you lose connection to your database for example, then it's handled appropriately.